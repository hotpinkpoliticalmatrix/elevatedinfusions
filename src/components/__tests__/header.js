import React from "react"
import renderer from "react-test-renderer"
import Header from "../header"

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Header siteTitle="Elevated Infusions Bakery" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
