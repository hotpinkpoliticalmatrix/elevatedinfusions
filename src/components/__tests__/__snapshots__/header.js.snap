// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary Header-appBar-1 mui-fixed MuiPaper-elevation4"
>
  <div
    className="MuiToolbar-root MuiToolbar-regular Header-container-4 MuiToolbar-gutters"
  >
    <div
      className="Header-flex-5"
    >
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-text Header-title-6"
        href="/#"
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        />
        <span
          className="MuiTouchRipple-root"
        />
      </a>
    </div>
    <div
      className="PrivateHiddenCss-smDown-23"
    />
  </div>
  <div
    className="PrivateHiddenCss-mdUp-25"
  />
</header>
`;
